util/csvtojson.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python
+
+import json, sys, csv, os, pipes, subprocess
```

Comment:
os, pipes, subprocess will be unnecessary once we take out the `deepdive sql` invocation.